packages/api/__tests__/specs/article.ts

```diff
@@ -22,7 +31,36 @@ beforeAll(async () => {
 })
 
 describe('Articles', () => {
-  describe('can be created/edited/deleted', () => {
+  describe('can be created/edited/deleted:', () => {
+    let articleIds: string[]
+    articleIds = []
```

Comment:
you can write this on the same line and use `const` since you only change the content of the array and not the array istelf.
`const articleIDs: string[] = []`

Reply:
Good to know!